catalog/installSystem.sh

```diff
@@ -13,23 +13,51 @@ cd "$SCRIPTDIR/../bin"
 
 echo Installing whisk.system entities.
 
-createPackage system
-createPackage util
-createPackage samples
+createPackage system -a description "Low-level OpenWhisk utilities"
+createPackage util -a description "Building blocks that format and assemble data"
+createPackage samples -a description "A suite of simple actions to help you get started with OpenWhisk"
 
 waitForAll
 
 install utils/pipe.js             system/pipe -a system true
 
-install utils/cat.js              util/cat -a description 'Concatenate array of strings'
-install utils/sort.js             util/sort -a description 'Sort array'
-install utils/head.js             util/head -a description 'Filter first K array elements and discard rest'
-install utils/date.js             util/date -a description 'Get current date and time'
+install utils/cat.js              util/cat -a description 'Concatenates input into a string' \
+                                           -a parameters '[ { "name": "lines", "required": true, "type": "array", "description": "An array of strings or numbers" } ]' \
+                                           -a sampleInput '{ "lines": [4, 2, 3] }' \
+                                           -a sampleOutput '{ "lines": [4, 2, 3], "payload": "4\n2\n3" }'
+
+install utils/sort.js             util/sort -a description 'Sorts an array' \
+                                           -a parameters '[ { "name": "lines", "required": true, "type": "array", "description": "An array of strings" } ]' \
+                                           -a sampleInput '{ "lines": [4, 2, 3] }' \
+                                           -a sampleOutput '{ "lines": [2, 3, 4], "length": 3 }'
+
+install utils/head.js             util/head -a description 'Extract prefix of an array' \
+                                           -a parameters '[ { "name": "lines", "required": true, "type": "array", "description": "An array of strings" }, { "name": "num", "required": false, "type": "integer", "description": "The length of the prefix" }]' \
+                                           -a sampleInput '{ "lines": [4, 2, 3], num: 2 }' \
+                                           -a sampleOutput '{ "lines": [4, 2], "num": 2 }'
+
+install utils/date.js             util/date -a description 'Current date and time' \
+                                           -a sampleOutput '{ "date": "2016-03-22T00:59:55.961Z" }'
+
+install samples/hello.js          samples/helloWorld -a description 'Demonstrates logging facilities' -a parameters '[{"name": "payload", "required":false, "description":"The string to be included in the log record"}]' \
+                                           -a sampleInput '{ "payload": "Cat" }' \
```

Comment:
we should change the parameter name in `hello.js` to be `name` rather than `payload` so it is consistent with `greeting`. this will require changing the unit tests (so in terms of scope, could be done separately).